build.gradle

```diff
@@ -168,12 +166,6 @@ project(':springboottrades-web') {
 		compile "org.springframework.boot:spring-boot-starter-thymeleaf:${springBootVersion}"
 		compile "org.springframework.boot:spring-boot-starter-security:${springBootVersion}"
 		compile "org.thymeleaf.extras:thymeleaf-extras-springsecurity4:${thymeleafExtras}"
-
-		//runtime
-		runtime "org.webjars:bootstrap:${bootstrapVersion}"
```

Comment:
Where are the bootstrap and query coming from? Spring Boot? If so do we need to control versions?